### ISSUE TITLE
Normalize input data

### ISSUE THREAD

**Issue by peerdavid** [peerdavid/ConvolutionalNeuralNetwork#5]

**Comment by peerdavid** [comment]
See http://stackoverflow.com/questions/33949786/how-could-i-use-batch-normalization-in-tensorflow

**Comment by peerdavid** [comment]
See issue #7
